components/contribution-flow/StepDetails.js

```diff
@@ -48,34 +49,37 @@ const StepDetails = ({ onChange, data, collective, tier, showFeesOnTop, router }
   const currency = tier?.amount.currency || collective.currency;
   const selectedInterval = data?.interval !== INTERVALS.flexible ? data?.interval : null;
 
+  const paymentMethods = [GQLV2_PAYMENT_METHOD_TYPES.CREDIT_CARD, GQLV2_PAYMENT_METHOD_TYPES.PAYPAL];
```

Comment:
Same here for variable name. Since we're using this in multiple places, having a common variable in `lib/constants/payment-methods.js` would also make sense.

Reply:
Done. 👍🏽